### ISSUE TITLE
Unable to build with Xcode 13 GM

### ISSUE THREAD

**Issue by jsorge** [kylef/PathKit#77]
When I build the package for release (using `swift build -c release`) I get the following errors:
```
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:594:12: error: value of optional type 'UnsafeMutablePointer<CChar>?' (aka 'Optional<UnsafeMutablePointer<Int8>>') must be unwrapped to a value of type 'UnsafeMutablePointer<CChar>' (aka 'UnsafeMutablePointer<Int8>')
      free(cPattern)
           ^
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:594:12: note: coalesce using '??' to provide a default when the optional value contains 'nil'
      free(cPattern)
           ^
                    ?? <#default value#>
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:594:12: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
      free(cPattern)
           ^
                   !
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:625:12: error: value of optional type 'UnsafeMutablePointer<CChar>?' (aka 'Optional<UnsafeMutablePointer<Int8>>') must be unwrapped to a value of type 'UnsafeMutablePointer<CChar>' (aka 'UnsafeMutablePointer<Int8>')
      free(cPattern)
           ^
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:625:12: note: coalesce using '??' to provide a default when the optional value contains 'nil'
      free(cPattern)
           ^
                    ?? <#default value#>
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:625:12: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
      free(cPattern)
           ^
                   !
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:626:12: error: value of optional type 'UnsafeMutablePointer<CChar>?' (aka 'Optional<UnsafeMutablePointer<Int8>>') must be unwrapped to a value of type 'UnsafeMutablePointer<CChar>' (aka 'UnsafeMutablePointer<Int8>')
      free(cPath)
           ^
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:626:12: note: coalesce using '??' to provide a default when the optional value contains 'nil'
      free(cPath)
           ^
                 ?? <#default value#>
/Users/ci/Downloads/PathKit-master/Sources/PathKit.swift:626:12: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
      free(cPath)
           ^
                !
```
Is this a straightforward patch that can be applied?

**Comment by ds8k** [comment]
Running into this same error with UnsafeMutablePointer

![Screen Shot 2021-09-21 at 5 19 17 PM](https://user-images.githubusercontent.com/4458812/134249193-9ca06dcf-605f-476d-9077-6b732a7f0036.png)

**Comment by urosMihailovic** [comment]
> Running into this same error with UnsafeMutablePointer
> 
> ![Screen Shot 2021-09-21 at 5 19 17 PM](https://user-images.githubusercontent.com/4458812/134249193-9ca06dcf-605f-476d-9077-6b732a7f0036.png)

Have same issue when trying to build

**Comment by Armenm** [comment]
Hi @kylef is this fix sufficient for the release?

**Comment by Armenm** [comment]
Oh sorry, didn't see the comment about releasing today. Thanks a lot

**Comment by Armenm** [comment]
🥳 https://github.com/kylef/PathKit/releases/tag/1.0.1

**Comment by kikeenrique** [comment]
Hi,
I'm still getting a compilation error with version 1.0.1


`~/Library/Developer/Xcode/DerivedData/ios-ci-helpers-dorskpgcnqncbihcmayjbhhgveqs/SourcePackages/checkouts/XcodeProj/Sources/XcodeProj/Extensions/Path+Extras.swift:30:18: Value of optional type 'UnsafeMutablePointer<CChar>?' (aka 'Optional<UnsafeMutablePointer<Int8>>') must be unwrapped to a value of type 'UnsafeMutablePointer<CChar>' (aka 'UnsafeMutablePointer<Int8>')
`
Anyone else?

**Comment by kylef** [comment]
> SourcePackages/checkouts/XcodeProj/Sources/XcodeProj/Extensions/Path+Extras.swift

This source file is not from PathKit

**Comment by kylef** [comment]
This has been resolved in 1.0.1 thanks to everyone who helped out here with propsed changes and investigations!

**Comment by kikeenrique** [comment]
Opps 😬, apologies, I'm really sorry.
